src/components/images_uploading/DisplayImageList.js

```diff
@@ -14,25 +14,23 @@ class DisplayImageList extends Component {
       <ul>
         {images.map(image => {
           return (
-            <>
-              <li key={image._id}>
-                Unit Number:
-                {image.unitNumber}
-                <Button
-                  className="m-2"
-                  href={`${process.env.REACT_APP_BASEURL}/images/${image.s3key}`}
-                >
-                  Show Floor Plan
-                </Button>
-                {/* The button below is currently doing nothing! */}
-                <Button
-                  className="m-2"
-                  href={`${process.env.REACT_APP_BASEURL}/images/${image.s3key}/annotate`}
-                >
-                  Add Annotation
-                </Button>
-              </li>
-            </>
+            <li key={image._id}>

```

Comment:
Remove the extra '<></>' here. The key issue is fixed.

Reply:
Awesome!